InfluxDb.cpp

```diff
@@ -137,6 +153,8 @@ boolean Influxdb::write(String data) {
     Serial.print(" --> writing to " + _db + ":\n");
     Serial.println(data);
   }
+  if(_db_v == 2)
+    http.addHeader("Authorization", "Token " + _token);
```

Comment:
Is it really necessary to add the header again as it is already added in line 104?

Reply:
Ah just saw you comment about `missing 'Token' header`.
Of course it makes sense to set the header for each request. 👍 

But then we can remove line 104.

Reply:
agreed